File: DSA/Stack_Queue_Deque/2390_.py
class Solution:
    def removeStars(self, s: str) -> str:
        #Make a list in which you will add or remove the element
        word=[]
        #Iterate through the s until the end
        for i in s:
        #if * come while iteration pop the elements else append it to the list
            if i=="*":
                word.pop()
            else:
                word.append(i)
        #Convert the list into string and then return it
        return "".join(word)

File: DSA/Stack_Queue_Deque/2390_.go
func removeStars(s string) string {
	stack := []byte{}
	for i := 0; i < len(s); i++ {
		if s[i] == '*' {
			if len(stack) > 0 {
				stack = stack[:len(stack)-1]
			}
		} else {
			stack = append(stack, s[i])
		}
	}

	return string(stack)
}